bitnami/redis/templates/sentinel/statefulset.yaml

```diff
@@ -589,6 +593,22 @@ spec:
             {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.extraVolumeMounts "context" $ ) | nindent 12 }}
             {{- end }}
         {{- end }}
+        {{- if .Values.sentinel.service.createMaster }}
+        - name: kubectl
+          image: {{ template "redis.kubectl.image" . }}
+          imagePullPolicy: {{ .Values.kubectl.image.pullPolicy | quote }}
+          command: {{- toYaml .Values.kubectl.command | nindent 12 }}
+          securityContext:
+            runAsUser: 0
+          volumeMounts:
+            - name: kubectl
+              mountPath: /kubectl
```

Comment:
We should rename this `kubectl` volume and also the mount path. It's confusing.
For instance, naming the volume `kubectl-shared` and mounting it at `/etc/shared` (or similar)